src/photon/photon_scheduler.c

```diff
@@ -23,8 +24,148 @@
 UT_icd task_ptr_icd = {sizeof(task *), NULL, NULL, NULL};
 UT_icd workers_icd = {sizeof(local_scheduler_client *), NULL, NULL, NULL};
 
+UT_icd pid_t_icd = {sizeof(pid_t), NULL, NULL, NULL};
+
 UT_icd byte_icd = {sizeof(uint8_t), NULL, NULL, NULL};
 
+void kill_process(pid_t pid) {
+  /* Kill the worker violently. */
+  kill(pid, SIGKILL);
+  waitpid(pid, NULL, 0);
```

Comment:
I'm a little nervous that in pathological situations this could cause the local scheduler to hang forever.

Even during normal usage, do we really want to local scheduler to block here?